mungegithub/BUILD

```diff
@@ -8,12 +8,14 @@ load(
 
 go_binary(
     name = "mungegithub",
+    features = ["-race"],
     library = ":go_default_library",
 )
 
 go_library(
     name = "go_default_library",
     srcs = ["mungegithub.go"],
+    features = ["-race"],
```

Comment:
You need to set this on the test itself that is failing.

Reply:
Yeah, I think you can set it on the package level but I just did that, I'll remove the extra ones.